saleor/wishlist/models.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import unicode_literals
+
+from uuid import uuid4
+
+from django.contrib.postgres.fields import HStoreField
+from django.db import models
+from django.core.urlresolvers import reverse
+from django.utils.encoding import python_2_unicode_compatible
+
+from saleor.core.utils import build_absolute_uri
+from ..product.models import ProductVariant, Product
+from ..userprofile.models import User
+
+
+@python_2_unicode_compatible
+class Wishlist(models.Model):
+    user = models.ForeignKey(User)
+    token = models.CharField(default=uuid4, editable=False, max_length=36,
```

Comment:
Can't we use `UUIDField` from Django? https://docs.djangoproject.com/en/1.10/ref/models/fields/#uuidfield